packages/common-all/src/api.ts

```diff
@@ -365,15 +364,19 @@ export class DendronAPI extends API {
     });
   }
 
-  configGet(req: WorkspaceRequest): Promise<APIPayload<ConfigGetPayload>> {
+  configGet(
```

Comment:
maybe I'm thinking about this wrong, but since `DCommonProps` has config removed, can't we get rid of these two methods altogether?

Reply:
Looks like the frontend still makes api calls to get the config (`slice.ts`)